stream-compositions/api/integrations-api/transaction-integration-api/src/main/resources/integration-api-v2.yaml

```diff
@@ -56,10 +56,16 @@ components:
     PullIngestionRequest:
       type: object
       properties:
-        userExternalId:
+        externalArrangementId:
```

Comment:
Shouldn't these request properties align with the request properties of the transaction-composition API?

We are missing `arrangementId` and `legalEntityInternalId`

Reply:
ArrangementId - Agreed and will be added in a follow-up PR.
LegalEntityId is not needed for the Integration Spec. It's only purpose of being in the composition service spec is for the Transaction Cursor

Reply:
Fixed